.gitignore

```diff
@@ -129,3 +129,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vim swap files
+**/*.swp
```

Comment:
This is fine, I guess, but my personal preference would be to have a small `.gitignore` with project specific content. For local setup, I use `.git/info/exclude`.

Is this approach with a huge .gitignore better for, say, using dev containers?

Reply:
Or a global .gitignore

In `~/.gitconfig`:

```
[core]
	excludesfile = /Users/bh2smith/.gitignore_global
```

which contains things like:

```
TODO.md
.venv/
.env.bkp
env/
venv/
.DS_Store
.idea/
.vscode/
out/

```

Reply:
> Is this approach with a huge .gitignore better for, say, using dev containers?

Not that I know of, I didn't know you could add it to .git/info/exclude. Thanks!